source/includes/_188-config-repos.md.erb

```diff
@@ -0,0 +1,5 @@
+# Pipeline as Code
+
+The Pipeline as Code API allows admin users to manage the config repositories.
```

Comment:
How bout something like `The config repo API allows admin users to define and manage config repos using which pipelines defined in external repositories can be included in GoCD, thereby allowing users to have their `**Pipeline as code**

Also I was wondering if we need to change the description of the Configuration Repository to something like, 
`GoCD's configuration is version controlled in a local git repository on the GoCD server, the Configuration Repository endpoint allows you to clone and fetch a copy of this repository.`